app/vmalert/group.go

```diff
@@ -163,6 +167,10 @@ func (g *Group) ID() uint64 {
 	hash.Write([]byte("\xff"))
 	hash.Write([]byte(g.Name))
 	hash.Write([]byte(g.Type.Get()))
+	hash.Write([]byte(g.Interval.String()))
+	if g.EvalOffset != nil {
+		hash.Write([]byte(g.EvalOffset.String()))
+	}
```

Comment:
I don't think this is needed. Why do you think we should do this?

Reply:
See https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4693#issuecomment-1646561507.